pkg/logs/logs.go

```diff
@@ -86,7 +87,7 @@ func Start() error {
 	// add the default sources after the AutoConfig has been executed once because
 	// we don't want the container_collect_all to reap all containers ownership
 	go func() {
-		common.BlockUntilAutoConfigRanOnce(time.Second * 30)
+		common.BlockUntilAutoConfigRanOnce(time.Millisecond * time.Duration(coreConfig.Datadog.GetInt("ac_load_timeout")))
```

Comment:
we can use `Datadog.GetDuration` here
```suggestion
		common.BlockUntilAutoConfigRanOnce(time.Millisecond * coreConfig.Datadog.GetDuration("ac_load_timeout"))
```

Reply:
🙇‍♂️